### DISCUSSION TITLE
How can I shorten the transcription sentences?

### DISCUSSION THREAD

**Discussion by Pimool** [openai/whisper#1627] [Q&A]
Hi, I'm using whisper to make a subtitle for videos.

The performance of whisper is really good, so that there are only few things to edit.

However, I thought some sentences are too long to add it into a video.

In this repository, it can be handled through passing MAX_LINE_WIDTH and MAX_LINE_COUNT through get_writer() method.

However, I am using whisper by calling through the openai which means

Not
```python
import whisper

whisper.load_model('')
# so on
```

Using
```python
import openai

transcript = openai.Audio.transcribe(model = 'whisper-1', file = 'audio.mp3', response_format = 'srt', language = 'ko')
with open('transcript.srt'), 'w', encoding = 'utf-8') as f:
    f.write(transcript)
```

How can I pass those parameters into this library?

**Comment by phineas-pta** [comment]
your question should be asked at: https://community.openai.com/tag/whisper

**Reply by Pimool** [reply]
I made a question there. Thanks!
